src/components/AddControlKey.svelte

```diff
@@ -103,7 +103,7 @@
     <li>Select the new control key from the dropdown list below.</li>
     <li>Click 'Add It.'</li>
     <li>This requires 3 signatures: two for the authorization payload, and one to send the transaction.</li>
-    <ul class="ml-6 list-disc">
+    <ul class="unordered-list">
```

Comment:
This seems unrelated to the FAQ PR.

Reply:
We are using the same styling in this file as we are in the answers for FAQ. Since the styling in the file above is now used in multiple paces, I created a class in app.css and applied the reusable class here.